.evergreen/run-tests.sh

```diff
@@ -6,6 +6,7 @@ set -eux
 /opt/python/3.10/bin/python3 -m venv venv
 . venv/bin/activate
 python -m pip install -U pip
+pip install ".[encryption]"
```

Comment:
Any objection to combining this line and the next line into `pip install -e .[encryption]`?

Reply:
OK filing under CI todo